Docs/11 DOM/README.md

```diff
@@ -1,160 +1,141 @@
-# DOM
+[F# Computation Expressions]: https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/computation-expressions
```

Comment:
We should put those reference links after the **#DOM Elements**, otherwise it is not rendered.

Maybe we should just call it DOM, otherwise the menu of docs will look that nice and clean.

Reply:
Will make the change